app/src/main/res/layout/hints_summary.xml

```diff
@@ -15,53 +15,49 @@
       type="org.oppia.android.app.hintsandsolution.HintsViewModel" />
   </data>
 
-  <LinearLayout
+  <androidx.constraintlayout.widget.ConstraintLayout
     android:id="@+id/hint_summary_container"
     android:layout_width="match_parent"
     android:layout_height="wrap_content"
-    android:layout_marginTop="32dp"
-    android:orientation="vertical">
+    android:layout_marginTop="32dp">
 
     <androidx.constraintlayout.widget.ConstraintLayout
```

Comment:
In general, when moving to ConstraintLayout we should try to get rid of layout nesting since that's one of the main benefits of ConstraintLayout. Do you see a way that we could get rid of this inner constraint layout & configure the views to still be laid out in the same way only with the other ConstraintLayout?

Reply:
Yes, that's a good idea. Updated accordingly.